packages/modules/imodel-browser/src/containers/ITwinGrid/ITwinTile.tsx

```diff
@@ -79,7 +81,10 @@ export const ITwinTile = ({
         thumbnail={
           <span
```

Comment:
No need to pass span here anymore.. 
` thumbnail={
          <ITwinIcon
            onClick={() => onThumbnailClick?.(iTwin)}
            style={{ cursor: onThumbnailClick ? "pointer" : "auto" }}
          />
        }` 
        Should be enough

Reply:
Updated